contracts/utils/cryptography/SignatureChecker.sol

```diff
@@ -47,4 +57,79 @@ library SignatureChecker {
             result.length >= 32 &&
             abi.decode(result, (bytes32)) == bytes32(IERC1271.isValidSignature.selector));
     }
+
+    /**
+     * @dev Verifies a signature for a given ERC-7913 signer and hash.
+     *
+     * The signer is a `bytes` object that is the concatenation of an address and optionally a key:
+     * `verifier || key`. A signer must be at least 20 bytes long.
+     *
+     * Verification is done as follows:
+     *
+     * * If `signer.length < 20`: verification fails
+     * * If `signer.length == 20`: verification is done using {isValidSignatureNow}
+     * * Otherwise: verification is done using {IERC7913SignatureVerifier}
+     *
+     * NOTE: Unlike ECDSA signatures, contract signatures are revocable, and the outcome of this function can thus
+     * change through time. It could return true at block N and false at block N+1 (or the opposite).
+     */
+    function isValidERC7913SignatureNow(
```

Comment:
I'm thinking that having 
- `isValidSignatureNow(address,bytes32,bytes)` 
- `isValidERC7913SignatureNow(bytes,bytes32,bytes)` 
might imply that the "default" version is the first one, and might push away from the second as an 7913 specific thing ... which it is ... but people not knowing 7913 might think it doesn't apply to them.

What if we named both of them the same, and relied on the input type to distinguish
- `isValidSignatureNow(address,bytes32,bytes)` 
- `isValidSignatureNow(bytes,bytes32,bytes)` 

The logic is to emphasize that they serve the same purpose, and that the second version is a generalisation of the first one.

Approving this and opening another issue/pr to discuss that